docs/howto/manage-charms.md

```diff
@@ -0,0 +1,68 @@
+(manage-charms)=
+# How to manage charms
+
+> See first: {external+juju:ref}`Juju | Build a charm <build-a-charm>`, {external+charmcraft:ref}`Charmcraft | Manage charms <manage-charms>`
+
+Hit the ground running with Ops by setting up standard project structure and tools.
+
+> See more: {external+charmcraft:ref}`Charmcraft | Manage Charmcraft <manage-charmcraft>`
+
+Use Charmcraft to quickly initialise your charm project. This generates the
+folder structure, creates placeholder configuration and code files, and
+configures development tooling.
+
+> See more:
+>
+> * {external+charmcraft:ref}`Charmcraft | Manage charms > Initialize a charm <initialise-a-charm>` (see also the best practice note on setting up a repository and considering your CI)
+> * [Charmcraft | Manage charms > Add charm project metadata, an icon, docs](https://canonical-charmcraft.readthedocs-hosted.com/en/latest/howto/manage-charms/#add-charm-project-metadata-an-icon-docs)
+
+<!--
+TODO: Add a reference link in charmcraft for the link above and the 'runtime details' one below, and switch over to external refs.
+-->
+
+The essence of a charm is the ``src/charm.py`` file. This is the entry point for
+your code whenever Juju emits an event, and defines the interface between Juju
+and the charm workflow.
```

Comment:
@dwilding I think this is where you'd add an initial bit about `src/{workload}.py`. We can do that in this PR if you like, or you could do it in a follow-up one later.

I think the main instructions for separating out the code go into the write and structure how-to, but a small mention here would be good.